src/citrine/resources/data_concepts.py

```diff
@@ -279,21 +281,50 @@ def get_json_support(cls):
             )
         return cls.json_support
 
+    def dump(self) -> dict:
```

Comment:
Is this calling `dump` from the inherited GEMD objects, and then just filling in the gaps with client specific fields in citrine python?

That seems logical for reconciling the different serialization strategies, but maybe I am interpreting this wrong.

Reply:
Yes, which is necessary so that when you copy a citrine-python reflection of a gemd-python object, you maintain the `audit_info` and `dataset` attributes as populated.